gui/src/app/view/hw.rs

```diff
@@ -26,14 +26,32 @@ pub fn hw_list_view<'a>(
                         HardwareWallet::Supported {
                             fingerprint,
                             version,
+                            registered,
                             ..
                         } => Row::new()
+                            .align_items(Alignment::Center)
                             .spacing(5)
                             .push(text(format!("fingerprint: {}", fingerprint)).small())
                             .push_maybe(
                                 version
                                     .as_ref()
                                     .map(|v| text(format!("version: {}", v)).small()),
+                            )
+                            .push_maybe(registered.and_then(|registered|
+                                    if !registered {
+                                        Some(Row::new()
+                                            .spacing(5)
+                                            .align_items(Alignment::Center)
+                                            .push(text("unregistered").small())
+                                            .push(
+                                                tooltip::Tooltip::new(
+                                                    icon::warning_icon(),
+                                                    "Policy is not registered on the device.\n You can register it in the settings.",
```

Comment:
```suggestion
                                                    "The wallet descriptor is not registered on the device.\n You can register it in the settings.",
```

We've been using "descriptor" and not policy elsewhere, let's be consistent.